Sources/MapboxNavigation/InstructionsCardViewController.swift

```diff
@@ -246,6 +246,7 @@ open class InstructionsCardViewController: UIViewController {
     
     @objc func orientationDidChange(_ notification: Notification) {
         instructionsCardLayout.invalidateLayout()
+        guard let itemCount = steps?.count, itemCount >= 0 && indexBeforeSwipe.row < itemCount else { return }
```

Comment:
There is the same check in the `snapToIndexPath` method that is the only other method that uses `handlePagingforScrollToItem` so it should be better to move this validation to `handlePagingforScrollToItem`.

Reply:
Thanks for the suggestion, I'll move it to the `handlePagingforScrollToItem`.